public/resources/stylesheets/page/_admin.scss

```diff
@@ -91,8 +101,58 @@
   font-weight: 600;
   background-image: linear-gradient(180deg, #0f40c8 0%, #0d4cb1 100%);
```

Comment:
save the colors in variables, and check if that color are not already in the variables file